cves/CVE-2015-6826.yml

```diff
@@ -236,8 +269,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: The CVE itself specifies a function in this library.
+  name: avcodec
```

Comment:
Was the vulnerable file also in this directory in the repository? Just curious if there is ever any sort of disconnect there

Reply:
Yes, the file was in the `libavcodec` directory.